src/Sylius/Bundle/ApiBundle/Tests/Serializer/ProductVariantNormalizerTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Tests\Serializer;
+
+use ApiPlatform\Api\IriConverterInterface;
+use Doctrine\Common\Collections\ArrayCollection;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Sylius\Bundle\ApiBundle\Serializer\ContextKeys;
+use Sylius\Bundle\ApiBundle\Serializer\ProductVariantNormalizer;
+use Sylius\Bundle\CoreBundle\SectionResolver\SectionProviderInterface;
+use Sylius\Component\Core\Calculator\ProductVariantPricesCalculatorInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+use Sylius\Component\Inventory\Checker\AvailabilityCheckerInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+final class ProductVariantNormalizerTest extends TestCase
```

Comment:
Should we add comment for development team to remove this and other new tests for sylius 2.0, since it's only testing deprecated feature?